blocks/user-licenses/user-licenses.js

```diff
@@ -1,5 +1,400 @@
+        backToProfileButton = createTag('a', { href: '/pages/profile', class: 'button primary', id: 'backToAccountButton' }, 'Back to Account');
+        licenseComponent.insertBefore(backToProfileButton, licenseComponent.firstChild);
+      }
+    } else {
+      section.remove();
+    }
+  });
+}
+
+async function displayLicenseDetails(event) {
+  hideOtherUserLicenseInformation();
+  const licenseId = event.target.getAttribute('data-license-id');
+  const userId = localStorage.getItem('auth0_id');
+  const accessToken = localStorage.getItem('accessToken');
+
+  if (userId == null || userId === undefined || accessToken == null || accessToken === undefined) {
+    window.location.assign(`${window.location.origin}`);
+  } else {
+    const data = await execute('dynamics_get_end_users_for_license', `&license_id=${licenseId}`, 'GET');
+    // DOM creation
+    const manageLicenseH2 = createTag('h2', '', `Manage License #${data.licenseid}`);
+    const licenseDetailsDiv = document.querySelector('.license-details');
+    licenseDetailsDiv.innerHTML = '';
+    licenseDetailsDiv.appendChild(manageLicenseH2);
+    const licenseDetailsDataDiv = createTag('div', { class: 'license-details-data' }, '');
+    licenseDetailsDiv.appendChild(licenseDetailsDataDiv);
+    const headings = ['License Administrator|_new_registereduser_value@OData.Community.Display.V1.FormattedValue', 'Account|_new_account_value@OData.Community.Display.V1.FormattedValue', 'Renewal Date|new_supportexpires@OData.Community.Display.V1.FormattedValue', 'Key Serial Number|new_licenseid', 'Product|_new_product_value@OData.Community.Display.V1.FormattedValue', 'License Type|zemax_seattype@OData.Community.Display.V1.FormattedValue', 'ZPA Support|new_premiumsupport@OData.Community.Display.V1.FormattedValue', 'Seat Count|new_usercount@OData.Community.Display.V1.FormattedValue', 'End User Count|new_endusercount@OData.Community.Display.V1.FormattedValue'];
+
+    let licenseDetailsRow = createTag('div', { class: 'license-details-row layout-33-33-33' }, '');
+    headings.forEach((heading, index) => {
+      const elementDetailCellDiv = createTag('div', { class: 'element-detail-cell' });
+      const elementDetailCellHeading = createTag('h3', { class: 'element-detail-cell-heading' }, heading.split('|')[0]);
+      const elementDetailCellDataPara = createTag('p', { class: 'element-detail-cell-data' }, data.license_detail[0][heading.split('|')[1]]);
+      elementDetailCellDiv.appendChild(elementDetailCellHeading);
+      elementDetailCellDiv.appendChild(elementDetailCellDataPara);
+      licenseDetailsRow.appendChild(elementDetailCellDiv);
+
+      if (index % 3 === 2) {
+        licenseDetailsDataDiv.appendChild(licenseDetailsRow);
+        licenseDetailsRow = createTag('div', { class: 'license-details-row layout-33-33-33' }, '');
+      }
+    });
+
+    const nickNameSetValue = (data.license_detail[0].zemax_nickname === undefined ? '' : data.license_detail[0].zemax_nickname);
```

Comment:
Usually `undefined` is not used in JS for comparisons. This can be simplified: 
```suggestion
    const nickNameSetValue = data.license_detail.[0].zemax_nickname ?? '';
```
The nullish coalescing operator (??) is then used to provide a default value of '' (empty string) if the left-hand side is null or undefined.